examples/ad738x_example.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2022 Analog Devices, Inc.
+#
+# SPDX short identifier: ADIBSD
+import adi
+import numpy as np
+
+ad738x_dev = adi.ad738x(uri="ip:analog", device_name="ad7381")
+ad738x_dev.rx_annotated = True
+ad738x_dev._rx_data_type = np.int32
```

Comment:
This should be automatically determined by the rx methods when the buffer metadata is read. Is that now happening?

Reply:
Removed that line, example seems to work properly.